enhancements/cluster-logging/cluster-logging-v2-apis.md

```diff
@@ -0,0 +1,515 @@
+          protocol:              # optional
+            type:                # enum: otelp 
+            version:             
+        kafka:
+          topic:           #templating?
+          brokers:
+          tuning:
+            maxWrite:            # quantity (e.g. 500k)
+            compression:         # enum of supported algos specific to the output
+        lokiStack:
+          tenantID:        #templating?
+          labelKeys:
+          tuning:
+            maxWrite:            # quantity (e.g. 500k)
+            compression:         # enum of supported algos specific to the output
+            minRetryDuration:
+            maxRetryDuration:
+        splunk:
+          index:           #templating?
+          tuning:
+            maxWrite:            # quantity (e.g. 500k)
+            compression:         # enum of supported algos specific to the output
+            minRetryDuration:
+            maxRetryDuration:
+        syslog:            #only supports RFC5424
+          severity:
+          facility:
+          trimPrefix:
+          tagKey:         #templating?
+          payloadKey:     #templating?
+          addLogSource:
+          appName:        #templating?
+          procID:         #templating?
+          msgID:          #templating?
+    status:
+      conditions:    # []metav1.conditions
+      inputs:        # map[string] metav1.conditions
+      outputs:       # map[string] metav1.conditions
+      filters:       # map[string] metav1.conditions
+      pipelins:      # map[string] metav1.conditions
+```
+
+
+Example:
+
+```yaml
+    apiVersion: "observability.openshift.io/v1"
+    kind: ClusterLogForwarder
+    metadata:
+      name: log-collector
+      namespace: acme-logging
+    spec:
+      inputs:
+      - name: infra-container
+        type: infrastructure
+        infrastructure:
+          sources: [container]
+      serviceAccount:
+        name: audit-collector-sa
+      pipelines:
+       - inputRefs:
+         - infra-container
+         - audit
+         outputRefs:
+         - rh-loki
+      outputs:
+      - name: rh-loki
+        type: lokiStack
+```
+
+This example:
+
+* Deploys a log collector to the 'acme-logging' namespace
+* Expects the administrator to have created a service account named 'audit-collector-sa' in that namespace
+* Expects the administrator to have bound the roles 'collect-audit-logs', 'collect-infrastructure-logs to the service account
+* Expects the administrator created a **LokiStack** CR named 'rh-loki' in the 'openshift-logging' namespace
+* Collects all audit log sources and only infrastructure container logs and writes them to the Red Hat managed lokiStack
+
+#### ConsoleLogViewPlugin CustomResourceDefinition:
+
+Following defines the API to support deployment of the console logging-view-plugin by the **cluster-logging-operator**.  It is intended for
+this resource to be obsoleted by the **cluster-observability-operator** in the near future.
```

Comment:
This is a good strategy for now, it may change based on timing of COO & logging releases  (e.g. if timing is right we may be able to drop the console plugin entirely) Need to co-ordinate with COO team as release approaches.

Reply:
Agree.  Hoping those details are solidified yesterday so we can remove this concept all-to-gether

Reply:
removed